.circleci/config.yml

```diff
@@ -96,6 +96,8 @@ jobs:
           version: "0.78.0"
       - hugo/hugo-build:
           asciidoc: true
+          htmlproofer-http-status-ignore: "'0,400,503,999'"
+          htmlproofer-timeframe: "'1y'"
```

Comment:
These parameters are in the `hugo/html-proofer` command, but not in the `hugo-build` command. Unsure if this is intentional.

Reply:
Yea - @jwflory, my original comment was all changes in this file. Sorry that was not clear.